keyboards/yetis/config.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2023 Norbert Pocs (norbertpocs0@gmail.com, jogme)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#pragma once
+
+/* Uncomment if you wish to use USB on the right half */
+#define MASTER_RIGHT
+//#define MASTER_LEFT
+
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
```

Comment:
This is ... very short and may be hard to actually trigger.  You may want to consider increasing the time here.

Reply:
This explains the almost impossible to get into the bootloader issue. (The value was copied from other keyboards.) Thanks, changed.